pkg/types/args_test.go

```diff
@@ -0,0 +1,92 @@
+package types_test
+
+import (
+	"reflect"
+
+	"github.com/appc/cni/pkg/types"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("UnmarshallableBool UnmarshalText", func() {
```

Comment:
Last thing I see - wrap all of the tests in some sort of top level Describe so you can avoid having this multiple types

```go
var _ = Describe(
```
Top level describe could just name the package you are testing.